### ISSUE TITLE
BinEnv is unable to find binaries in global mode.

### ISSUE THREAD

**Issue by heavyblack1** [devops-works/binenv#227]
I installed `binenv` according to your system-wide installation instructions. And also add `echo "export BINENV_GLOBAL=true" >> .bashrc` and `exec $SHELL`.

I tried running `sudo bat`, and it didn't work.

```
2023-02-07T12:41:26+01:00 INF updating distribution list
2023-02-07T12:41:26+01:00 ERR no versions found for distribution "bat" (from bat). Something is really odd.
2023-02-07T12:41:26+01:00 FTL binenv: unable to find proper version for bat ()
```

After some digging, I returned from system-wide installation to the main `readme.md` and found the missing answer to my problem.

`sudo env "PATH=$PATH" "HOME=$HOME" bat`

```
2023-02-07T13:04:15+01:00 ERR no versions found for distribution "bat" (from bat). Something is really odd.
2023-02-07T13:04:15+01:00 FTL binenv: unable to find proper version for bat ()
```

But even this didn't solve issue.

Is this a bug? Or I didn't understand your guide?

Thanks for help

**Comment by leucos** [comment]
Canyou confirm that  `bat` installaion went well with `binenv install -g bat` ?

Did you try `sudo env "PATH=$PATH" "HOME=$HOME" "BINENV_GLOBAL=true" bat`  ?

Could you also try `sudo -E bat` also ?

**Comment by heavyblack1** [comment]
I looked in to history. Yes i specified -g option.  And other two command suggested just hanged.

**Comment by leucos** [comment]
Thanks a lot for the report. This is now solved in v0.19.8.
